source/_components/device_tracker.geofency.markdown

```diff
@@ -10,37 +10,14 @@ footer: true
 logo: geofency.png
 ha_category: Presence Detection
 ha_release: 0.53
+ha_iot_class: "Cloud Push"
 ---
 
-This platform allows you to detect presence using [Geofency](http://www.geofency.com/). Geofency is a [paid app](https://itunes.apple.com/app/id615538630) for iOS that lets users to configure a request that will be sent when a geofence or iBeacon region is entered or exited. This can be configured with Home Assistant to update your location.
+This platform allows you to detect presence using [Geofency](http://www.geofency.com/).
 
-To integrate Geofency in Home Assistant, add the following section to your `configuration.yaml` file:
-
-```yaml
-# Example configuration.yaml entry
-device_tracker:
-  - platform: geofency
-```
-
-{% configuration %}
-mobile_beacons:
-  description: List of beacon names that are to be treated as *mobile*. The name must match the name you configure in Geofency. By default, beacons will be treated as *stationary*.
-  required: false
-  type: list
-{% endconfiguration %}
-
-A full sample configuration for the `geofency` platform is shown below:
-
-```yaml
-# Example configuration.yaml entry
-device_tracker:
-  - platform: geofency
-    mobile_beacons:
-      - car
-      - keys
-```
-
-To configure Geofency, you must configure (via the Webhook feature) to send a POST request to your Home Assistant server at `http://<ha_server>/api/geofency`. Use the default POST format, and make sure to include the API password if you have configured a password in Home Assistant (add `?api_password=<password>` to the end of the URL). Make sure to enable the 'Update Geo-Position' functionality for mobile beacons.
+<p class='note'>
+You must have the [Geofency component](/components/geofency/) configured to use this device tracker.
```

Comment:
`Geofency component` -> `Hub` or `Platform` I guess?

Reply:
Geofency will now be a component, so that sounds right. It is definitely not a `Hub`.

Reply:
Left a comment in the other file. The linked file is categorized as "Hub", enhance my comment on this line.

Reply:
Changed other file.